accounts/templates/registration/signup.html

```diff
@@ -0,0 +1,16 @@
+<!-- templates/signup.html -->
+{% extends 'base.html' %}
+
+
+{% if error_messages %} error_messages {% endif %}
```

Comment:
- When you visit the signup form for the first time, a lot of errors show up.
- I think that we could use the bootstrap forms for better UI.

Reply:
Suppressed help messages for the username, password1, password2 field which appeared.

Also i'll leave the UI stuff to you :))